src/_locales/en_US/messages.json

```diff
@@ -279,6 +279,10 @@
             }
         }
     },
+    "popup_info_youtube": {
+        "message": "Experiencing issues with YouTube? Are ads playing? It's unlikely that Privacy Badger is the problem. ",
+        "description": "Message displayed to users on YouTube to clarify misconceptions that lead to high numbers of YouTube error reports."
```

Comment:
I think it's enough to just describe where the string can be found. Like, "Message in the popup when you visit YouTube.com"

Reply:
Good to know, will do!